wandb/sdk/interface/interface_shared.py

```diff
@@ -382,9 +382,9 @@ def _publish_use_artifact(self, use_artifact: pb.UseArtifactRecord) -> Any:
         rec = self._make_record(use_artifact=use_artifact)
         self._publish(rec)
 
-    def _communicate_artifact(self, log_artifact: pb.LogArtifactRequest) -> Any:
+    def _deliver_artifact(self, log_artifact: pb.LogArtifactRequest) -> Any:
```

Comment:
Nit: let's add proper type here.
```suggestion
    def _deliver_artifact(self, log_artifact: pb.LogArtifactRequest) -> MailboxHandle:
```

Reply:
fixed, and a few other ones I saw